packages/cli/src/commands/org/resources/pull.ts

```diff
@@ -33,6 +34,29 @@ import {SnapshotFactory} from '../../../lib/snapshot/snapshotFactory';
 import {confirmWithAnalytics} from '../../../lib/utils/cli';
 import {spawnProcess} from '../../../lib/utils/process';
 
+const PullCommandStrings = {
+  projectOverwriteQuestion: (
+    resourceFolderName: string
+  ) => dedent`There is already a Coveo project with resources in it.
+  This command will overwrite the ${resourceFolderName} folder content, do you want to proceed? (y/n)`,
+  resourcePullQuestion: (
+    currentOrgId: string,
+    pullOrgId: string
+  ) => dedent`You are currently connected to the ${formatOrgId(
+    currentOrgId
+  )} organization, but are about to pull resources from the ${formatOrgId(
+    pullOrgId
+  )} organization.
+      Do you wish to continue? (y/n)`,
```

Comment:
```suggestion
      Do you wish to continue with `${formatOrgId(pullOrgId}` ? (y/n)`,
```
@olamothe ?

Reply:
My comment is more to the effect that as an end user, I would be asking questions like: 

"So what's the big deal if I'm connected to X but pull from Y? Am I doing something stupid if the CLI is asking me to confirm ?"

Reply:
Mhh, you had a good hunch here :D 

This prompt was done prior to https://github.com/coveo/cli/pull/708 where we changed the precondition so that it checked the orgId of the SPM if an SPM was provided. 

So in fact, this is 'useless' code, because it was meant to warn the user that I may not have access to the org.